package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "type": "module",
   "scripts": {
+    "ci": "yarn install --prefer-offline --frozen-lockfile",
```

Comment:
Legger til denne kommandoen for å kunne bruke samme workflow for prosjekter som bare har bare frontend, og prosjekter som har både frontend og backend